test/widgets/inbox_test.dart

```diff
@@ -51,25 +51,46 @@ void main() {
     await tester.pumpAndSettle();
   }
 
+  List<StreamMessage> generateStreamMessages(
+    ZulipStream stream,
+    int count,
+  ) =>
+      List.generate(
```

Comment:
nit: much too long for `=>`  syntax — use braces:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using-for-short-functions-and-methods

Reply:
Done.